internal/awsservice/eks.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package awsservice
+
+import (
+	_ "embed"
```

Comment:
Do we need to import this? I don't see an embed in this file

Reply:
No, will remove. Had the embed here originally but moved it out

Reply:
... good catch. I didn't even think about why I didn't see a new file to review the second time around